pkg/operator/ceph/controller/controller_utils.go

```diff
@@ -116,6 +118,22 @@ func LoopDevicesAllowed() bool {
 	return loopDevicesAllowed
 }
 
+func SetEnforceHostNetwork(data map[string]string) {
+	Strval := k8sutil.GetValue(data, EnforceHostNetworkSettingName, EnforceHostNetworkDefaultValue)
+	var err error
+	var val bool
+	if val, err = strconv.ParseBool(Strval); err != nil {
```

Comment:
nit: No need to declare the vars on different lines above if you separate the `if` statement like this. Either way works, but IMO this is the more common pattern with golang
```suggestion
	val, err := strconv.ParseBool(Strval)
	if err != nil {
```